ww

```diff
@@ -15,8 +15,14 @@ activate() {
   if [ -z "$sourced" ]; then
     errorOut "use 'source ww $1'"
   fi
-  
-  workspace="${WW_DEFAULT_PATH}/${1}"
+
+  # If provided workspace is a absolute path, use it as so. If not, then make
```

Comment:
But let you know this is not how it is done on windows.

On windows if you pass any path (absolute or relative) to a valid workspace it "activates" that workspace.

```
for %%i in (%WW_DEFAULT_VOLUMES%) do (
    if not exist %_WW_WORKSPACE%\%WW_PROJECTS_SUBDIR% (
        if exist %%i:\%_WW_WORKSPACE%\%WW_PROJECTS_SUBDIR% (
            set _WW_WORKSPACE=%%i:\%_WW_WORKSPACE%
        )
    )
)
...
REM Change WW_CURRENT_WORKSPACE to absolute PATH, if it is still relative
for /F "tokens=* delims=\" %%i in ("%_WW_WORKSPACE%") do set "WW_CURRENT_WORKSPACE=%%~fi"
```

The check is nested inside the for loop (not the best place for it to be, I agree on that).
The "valid"  check is "if arg1\Projects exist".

Reply:
Updated to mimic the Windows behaviour :)

Reply:
*ping* could you re-review if the mimic was properly done?